setup.py

```diff
@@ -71,11 +62,16 @@
     url="https://github.com/python-trio/trio-asyncio",
     license="MIT -or- Apache License 2.0",
     packages=["trio_asyncio"],
-    install_requires=install_requires,
+    install_requires=[
+        "trio >= 0.22.0",
+        "outcome",
+        "sniffio",
+        "exceptiongroup >= 1.0.0rc9; python_version < '3.11'",
```

Comment:
Best not to specify a RC version, as this causes pip to install a pre-release when available.